apollo-router/src/router_factory.rs

```diff
@@ -297,17 +233,9 @@ mod test {
     impl Plugin for AlwaysFailsToStartPlugin {
```

Comment:
Maybe rename now that it's always fail to create?
```suggestion
    impl Plugin for AlwaysFailsToCreatePlugin {
```